components/apps.js

```diff
@@ -545,37 +561,28 @@ SteamUser.prototype._getLicenseInfo = async function() {
 /**
  * Get list of appids this account owns. Only works if enablePicsCache option is enabled and appOwnershipCached event
  * has been emitted.
- * @param {boolean} [excludeSharedLicenses=false] - Pass true to exclude licenses that we have through family sharing
+ * @param {object} options - Options for what counts for ownership
  * @returns {int[]}
  */
-SteamUser.prototype.getOwnedApps = function(excludeSharedLicenses) {
-	if (!this.options.enablePicsCache) {
-		throw new Error("PICS cache is not enabled.");
-	}
+SteamUser.prototype.getOwnedApps = function(options) {
+	this._ensurePicsCache();
 
-	if (!this.picsCache.packages) {
-		throw new Error("No data in PICS package cache yet.");
-	}
-
-	let ownedPackages = this.getOwnedPackages(excludeSharedLicenses);
+	let ownedPackages = this.getOwnedPackages(options);
 	let appids = {};
 
 	ownedPackages.forEach((pkg) => {
 		if (!this.picsCache.packages[pkg]) {
+			this.emit('debug', `Failed to get owned apps for package ${pkg.packageid}`);
```

Comment:
This probably shouldn't happen, so log it to debug. Same goes for L581, L623, L629, L700 and L706.

Reply:
Could use `this._warn` instead if it's not supposed to happen but could have negative implications if it does.

Reply:
Hmm, but why are you not doing that for other errors, like https://github.com/DoctorMcKay/node-steam-user/blob/9fe5a45cb3b6a93246c74b270ed51e888d6d692d/components/apps.js#L433

Reply:
That particular error would just result in a delay in when the changelist is detected and emitted. It wouldn't have the potential to corrupt data returned to the consumer application.